src/graphics/color.h

```diff
@@ -22,6 +22,7 @@ typedef uint32_t color_t;
 #define COLOR_NO_MASK 0xffffff
 #define COLOR_MASK_RED 0xff0818 // bit lost 0xf863
 #define COLOR_MASK_GREEN 0x18ff18
+#define COLOR_ALPHA_BLUE 0x665555ff
```

Comment:
I played around a bit with the colors, since this one looks purple, especially on desert maps.
```suggestion
#define COLOR_ALPHA_BLUE 0x663377ff
```
Current color:
![color-old](https://user-images.githubusercontent.com/104874/75851909-15a42c80-5deb-11ea-9d41-896a3bf3f183.png)

New color:
![color-new](https://user-images.githubusercontent.com/104874/75851921-1dfc6780-5deb-11ea-87e0-759e4829da6e.png)